package.json

```diff
@@ -9,33 +9,83 @@
 		"theme": "dark"
 	},
 	"license": "SEE LICENSE IN LICENSE",
-	"version": "1.1.3",
+	"version": "1.2.0",
 	"engines": {
 		"vscode": "^1.82.0"
 	},
 	"categories": [
-		"Snippets", "Programming Languages", "Other"
+		"Snippets",
+		"Programming Languages",
+		"Other"
+	],
+	"activationEvents": [
+		"onStartupFinished"
 	],
-	"activationEvents": ["onStartupFinished"],
```

Comment:
Curious removal. Are there any implications?

I distinctly recall this fixing a particular bug.

Reply:
Actually, here is exactly why this is needed: https://stackoverflow.com/questions/56086972/how-do-i-run-an-extension-command-on-startup-on-vscode

This trick is used to actually initialise the extension's configuration of settings.

Reply:
Oh, not sure why I actually did this. Rolling back.

Reply:
Nevermind. It's just indented 🤣 
![image](https://github.com/devSparkle/overture-vscode/assets/4990534/81b8c037-8ebc-4dab-9da7-6cccb0e56cdd)